en/developer_guide/logs.rst

```diff
@@ -0,0 +1,89 @@
+Logs
+====
+
+For logs we use Monolog. The Logs table is displayed in administration section in the backoffice.
+It displays the date and time, user's IP, user's name, the current site's name and the log's message.
+
+The LogBundle is in openorchestra-cmsbundle.
+
+Configuration
+-------------
+
+The logs's configuration is in app/config/log.yml. This file allows to declare the handlers.
+The handlers configure where save the logs the information type to record and channels.
```

Comment:
Your description is a little too procedural I think, you should rather start by the end : 

Monolog uses channels to record the log. To link a channel and a record, you will have to define an handler. The configuration of the handler can be placed in the ``app/config/log.yml`` file.